tsconfig.json

```diff
@@ -1,18 +1,33 @@
 {
   "compilerOptions": {
-    "baseUrl": "src",
-    "rootDir": "./",
-    "outDir": "build",
-    "target": "esnext",
+    // Basic Options
+    "target": "ES2017",
     "module": "commonjs",
-    "moduleResolution": "node",
+    "declaration": true,
     "sourceMap": true,
-    "removeComments": true,
+
+    // Directories
+    "outDir": "build",
+
+    // Strict Type-Checking Options
     "strict": true,
-    "resolveJsonModule": true,
+    "noImplicitAny": false,
+
+    // Additional Checks
+    "noUnusedLocals": false, // TODO KEEFER make this TRUE after implementing getMinimumFee in Xpring-Client
```

Comment:
Make an Asana task for this?

Reply:
https://app.asana.com/0/1143722888069470/1159494420164193